cellbrowser_tools/fov_processing.py

```diff
@@ -657,10 +928,22 @@ def _save_and_post(
         if image is not None:
             transposed_image = image.transpose(1, 0, 2, 3)
             log.info("saving image...")
+            # calculate the proper plane count
+            omepixels = omexml.images[0].pixels
+            check_num_planes(omepixels)
+            omepixels.tiff_data_blocks = [
+                TiffData(
+                    plane_count=omepixels.size_c * omepixels.size_z * omepixels.size_t
+                )
+            ]
+
+            ome_str = to_xml(omexml)
+            # appease ChimeraX and possibly others who expect to see this
+            ome_str = '<?xml version="1.0" encoding="UTF-8"?>' + ome_str
```

Comment:
possibly we should file an issue with ome-types regarding whether this could be an option in to_xml

Reply:
Done: https://github.com/tlambert03/ome-types/pull/69